componentLibraries/defaultLibrary/Signal/Control/SignalPID2.hpp

```diff
@@ -85,7 +87,16 @@ class SignalPID2 : public ComponentSignal
         {
             addWarningMessage("Tt is lower then Ti this is not correct!");
         }
+        if (mUmin > mUmax)
+        {
+            addErrorMessage("Umin must be <= Umax!");
+        }
+        if (mUoutmin > mUoutmax)
+        {
+            addErrorMessage("Uoutmin must be <= Uoutmax!");
+        }
```

Comment:
Good idea to add these checks. Maybe we should also check that Umin/max lies between the bound of Uoutmin/max. Because I do not think that you ever want to have Umax to ba larger then Uoutmax, that would cause windup risk. What do you think@robbr48

Reply:
I didn't add it so that you could disable anti-windup if you, for some reason I cannot come up with , want. Maybe Warning or Info message rather than Error? We may also silently do e.g. `mUmax=min(mUmax, mUoutmax)` plus a message. Just let me know what you prefer.

Reply:
Next to naming, deciding this sort of thing is the most difficult thing. First I was think that we should silently limit the value internally, but then that would be strange, if parameters you have set, or not, would be ignored. Lets just leave it in the current state (no further change needed), after-all it is a simulation of a controller and you may wast to do strange things.